packages/nexus-sdk/src/Schema/__tests__/schema.spec.ts

```diff
@@ -4,6 +4,11 @@ import { SchemaPayload } from '../types';
 
 const schema = Schema(mockFetchers, { uri: 'http://api.url/v1' });
 
+const responseTriples = `_:ec503c665478101a9c4f75845d20c20e <http://www.w3.org/ns/shacl#minCount> "1"^^<http://www.w3.org/2001/XMLSchema#integer> .
```

Comment:
I guess this is kind of useless now... could be any string. I'm just thinking about coming back to this test in a few months and thinking "why do we have such a specific string"